build.sbt

```diff
@@ -28,7 +28,8 @@ lazy val commonSettings = Seq(
   testOptions in Test += Tests.Argument(TestFrameworks.ScalaTest, "-h", "target/test-reports"),
   coverageEnabled in Test := true,
   coverageMinimum := 60,
-  coverageFailOnMinimum := true
+  coverageFailOnMinimum := true,
+  coverageExcludedPackages := "<empty>;cromwell\\.pipeline.*"
```

Comment:
Something is wrong. I don't see a lot of module here
https://coveralls.io/builds/39858444
Compare this to another report
https://coveralls.io/builds/39810273